src/schema/v2/__tests__/filterArtworksConnection.test.ts

```diff
@@ -619,4 +620,63 @@ describe("artworksConnection", () => {
       ])
     })
   })
+
+  describe("filter by marketing_collection_id", () => {
+    beforeEach(() => {
+      const filterArtworksLoader = jest.fn((args) => {
+        if (args.marketing_collection_id === "kaws-toys") {
+          return Promise.resolve({
+            hits: [
+              {
+                id: "kaws-toys",
+              },
+            ],
+            aggregations: {
+              total: {
+                value: 1,
+              },
+            },
+          })
+        }
+
+        return Promise.reject("unexpected marketing_collection_id")
+      })
+
+      context = {
+        authenticatedLoaders: {},
+        unauthenticatedLoaders: {
+          filterArtworksLoader: filterArtworksLoader,
+        },
+      }
+    })
+
+    it("returns correct artwork", async () => {
+      config.ENABLE_GRAVITY_MARKETING_COLLECTIONS = true
+
+      const query = gql`
+        {
+          artworksConnection(
+            input: {
+              marketingCollectionID: "kaws-toys"
+              first: 1
+              after: ""
+              aggregations: [TOTAL]
```

Comment:
#minor: do we need these? I think artworksConnection should work fine without them, but let me know if I'm wrong!

Reply:
removed, we don't need it, thanks!